spec/index.bs

```diff
@@ -1375,6 +1375,7 @@ enum GPUFeatureName {
     "pipeline-statistics-query",
     "texture-compression-bc",
     "timestamp-query",
+    "minmax-sampling",
```

Comment:
Is minmax sampling allowed for uint, int textures, but also for comparison samplers?

Reply:
The minmax feature in vulkan doesn't require support for any integer formats, so I assume we wouldn't support any. It also can't be used with comparison samplers apparently (kinda weird).

> If the format is a depth/stencil format, this bit only specifies that the depth aspect (not the stencil aspect) of an image of this format supports min/max filtering, and that min/max filtering of the depth aspect is supported when depth compare is disabled in the sampler.

Reply:
It could be, but getting the stats for this is hard :(
For now the spec only allows it on a few selected formats (see last commit).

Reply:
Wow, that's not a lot of formats.